docs/uml/main/ExtEvSimulationClasses.puml

```diff
@@ -164,10 +164,10 @@ package simona-api {
     ActivationMessage --|> ExtTrigger
 
     interface ExtTriggerResponse
-    class CompletionMessage {
+    class Completion {
         - List<Long> newTriggers
```

Comment:
This is also not up-to-date anymore. Check out `edu.ie3.simona.ontology.messages.SchedulerMessage.Completion`

Reply:
Do i update the  UML class like this?
class Completion{
 - ActorRef Activation
 - Option Long(tick)
}
or how do i translate the statements 

actor: ActorRef[Activation],
newTick: Option[Long] = None,

into UML?

Reply:
Think it should work like this (documentation [here](https://plantuml.com/en/class-diagram)):

```plantuml
class Completion {
    + ActorRef<Activation> actor
    + Option<Long> newTick
}
```